shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/Alert.kt

```diff
@@ -203,4 +205,64 @@ data class Alert(
     fun anyInformedEntity(predicate: (InformedEntity) -> Boolean) = informedEntity.any(predicate)
 
     fun matchingEntities(predicate: (InformedEntity) -> Boolean) = informedEntity.filter(predicate)
+
+    companion object {
+        /**
+         * Filter the list of alerts to only the ones that match the given filter parameters and
+         * affect boarding.
+         */
+        fun filter(
+            alerts: List<Alert>,
+            directionId: Int?,
+            routeIds: List<String>,
+            stopIds: Set<String>
+        ): List<Alert> {
+            return applicableAlerts(
```

Comment:
I'm a bit confused by this function, is this to accommodate a common interface with some other model? Looking at it without context it seems redundant.

Reply:
Ah yep good point - this should replace that. Moving it to Alert felt like a better fit since `applicableAlerts` is not RoutePattern specific